sysid/modelo.m

```diff
@@ -0,0 +1,279 @@
+function [sys,fit] = modelo(arquivo, parametros_arx, plano, aquisicao)
+
+%Carrega vetores dos arquivos .mat
+A = load(arquivo,'mat_d');
+currdata = A.mat_d.currdata;
+orbx = double(A.mat_d.orbx);
+orby = double(A.mat_d.orby);
+sampling_frequency = A.mat_d.sampling_frequency;
+prbs_n = A.mat_d.prbs_lfsr_len(1);
+prbs_duration = A.mat_d.prbs_step_duration(1);
+moving_avg = double(A.mat_d.prbs_mov_avg_taps);
+
+%Remoção de transiente
+currdata = currdata(19051:end, :);
+orbx = orbx(19051:end, :);
+orby = orby(19051:end, :);
+
+%Separação de conjunto de estimação e validação
+h1=length(currdata);
+h2=int32(length(currdata)/2);
+currdata = currdata(1:h2);
+orbx = orbx(1:h2);
+orby = orbx(1:h2);
+
+%Vetores para plot de validação
+%{
+currdata_val = currdata(h2:h1);
+orbx_val = orbx(h2:h1);
+orby_val = orby(h2:h1);
+%}
+
+%Redução do tempo de aquisição (opcional)
+T_aq=single(h2)/sampling_frequency;
+if exist('aquisicao', 'var')
```

Comment:
I think it'd be better if this argument was the number of samples to take into account for the model fitting. We gain nothing but some inaccuracy by abstracting it to time...